design/src/App.scss

```diff
@@ -77,6 +78,115 @@ $hover-color: #f8cc80;
     }
 }
 
+.glogg-h1 {
+    font-family: 'Merienda', cursive;
+    font-size: 35pt;
+    color: #ffffff;
+    text-align: center;
+}
+
+.glogg-row1{
```

Comment:
Have you decided on a code style, whether to have a space after the selector and before the curly brace, or not? Just a minor detail :)

Reply:
changed it with space